config/blacklight.yml

```diff
@@ -1,8 +1,9 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: <%= ENV['SOLR_DEVELOPMENT_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
 test: &test
   adapter: solr
+  url: <%= ENV['SOLR_TEST_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_TEST_PORT', 8985)}/solr/hydra-test" %>
```

Comment:
Why support both a `SOLR_DEVELOPMENT_URL` and a `SOLR_DEVELOPMENT_PORT` env vars? Is the port version really needed? Why not just always opt for the URL version if you are planning not to use the default settting?